### PR TITLE
In progress

### PR DESCRIPTION
**Pull request by onekatytocode**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,29 +3,33 @@
 <head>
 	<title>Home page</title>
 	<link rel="stylesheet" href="style.css" type="text/css" />
+	<link rel="stylesheet" href="navbar.css" type="text/css" />
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<link rel="preconnect" href="https://fonts.gstatic.com">
 	<link href="https://fonts.googleapis.com/css2?family=Raleway:wght@200&display=swap" rel="stylesheet">
+	<script src="https://kit.fontawesome.com/a5bc471788.js" crossorigin="anonymous"></script>
```
Comment (nacm1269) [inline index.html:12]: Maybe add the normalize css file 

<link rel="stylesheet" href="normalize.css" type="text/css">

File: index.html [hunk 2]
```diff
@@ -3,29 +3,33 @@
...
 				
 				<li><a href="#about">ABOUT ME</a></li>
-				<li><a href="portfolio.html">PORTFOLIO</a></li>
-				<li><a href="contact.html">CONTACT</a></li>	
+				<li><a href="#portfolio">PORTFOLIO</a></li>
+				<li><a href="#contact">CONTACT</a></li>	
 				
 			</ul>
+			
+			<p class="katy">KATY EVANS</p>
```
Comment (nacm1269) [inline index.html]: You could make this a h1/h2

File: index.html [hunk 3]
```diff
@@ -3,29 +3,33 @@
...
 	<link href="https://fonts.googleapis.com/css2?family=Raleway:wght@200&display=swap" rel="stylesheet">
+	<script src="https://kit.fontawesome.com/a5bc471788.js" crossorigin="anonymous"></script>
 	
 </head>
 	<body>
 		<nav>
-			<div class="item2">KATY EVANS</div><div class="aspire">aspiring full-stack web developer</div>
 			<ul>
 				
 				<li><a href="#about">ABOUT ME</a></li>
```
Comment (nacm1269) [inline index.html:18]: To change the nav links colour to white use  `li a { color: #ffffff; }` I think you just have li targeted atm

File: index.html [hunk 4]
```diff
@@ -3,29 +3,33 @@
...
+				<li><a href="#contact">CONTACT</a></li>	
 				
 			</ul>
+			
+			<p class="katy">KATY EVANS</p>
 		</nav>
-		
+		<div class="aspire">aspiring full-stack web developer</div>
 		
 		<div class="contain"><img src="thissign.jpg" alt="neon this is the sign you've been looking for"></div>
```
Comment (nacm1269) [inline index.html:28]: Good use of alt tag

File: index.html [hunk 5]
```diff
@@ -3,29 +3,33 @@
...
 		
 		
 		
 				<section class="about">
```
Comment (nacm1269) [inline index.html:32]: The background colour here doesn't quite stretch to cover the section

File: index.html [hunk 6]
```diff
@@ -37,14 +41,30 @@
 			<div><h3>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</h3></div>
 				</section>
 		
-		
-		
+
+	<article id="portfolio">
```
Comment (nacm1269) [inline index.html:47]: As you're working on this atm, not all of the comments to follow may be relevant as i'm sure you'll sort as you get to it

File: index.html [hunk 7]
```diff
@@ -37,14 +41,30 @@
...
+		<h1>Portfolio projects</h1>
```
Comment (nacm1269) [inline index.html:48]: Font colour is white on a white background

File: index.html [hunk 8]
```diff
@@ -37,14 +41,30 @@
...
+			<div class="container">
```
Comment (nacm1269) [inline index.html:49]: Looks good here, as you add projects you may want to use flex-wrap to make easy to view in mobile

File: index.html [hunk 9]
```diff
@@ -37,14 +41,30 @@
...
+			<div class="3"><img src="pilot.png"></div>	
+			</div>
+	</article>
+
+
+	<div class="contact">
+		<ul class="contact-list">
+		<a href="https://github.com/onekatytocode"><i class="fab fa-github"></i></a>
+		<a href="1katytogo@gmail.com"><i class="far fa-envelope"></i>
+		<a href="hhtp://www.linkedin.com/in/katy-evans-8589171b8"><i class="fab fa-linkedin-in"></i>
```
Comment (nacm1269) [inline index.html]: link doesn't seem to work

File: mainstyle.css [hunk 10]
```diff
@@ -3,42 +3,28 @@
 }
 
 body {
-	background-color: #986960;
+	background-color: #000000;
```
Comment (nacm1269) [inline mainstyle.css:6]: Do you need two background colours?

File: index.html [hunk 11]
```diff
@@ -3,29 +3,33 @@
...
-		
+		<div class="aspire">aspiring full-stack web developer</div>
 		
 		<div class="contain"><img src="thissign.jpg" alt="neon this is the sign you've been looking for"></div>
 		
 		
 		
 				<section class="about">
+					<h2>Where I am right now..</h2>
 					<div class="item">Currently at iO Academy in beautiful Bath studying the Full-Stack track training.
```
Comment (nacm1269) [inline index.html]: Could use more semantic class name

File: index.html [hunk 12]
```diff
@@ -37,14 +41,30 @@
...
 		
-		
-		
+
+	<article id="portfolio">
+		<h1>Portfolio projects</h1>
+			<div class="container">
+			<div class="1"><img src="pilot.png"></div>
+			<div class="2"><img src="jt.png"></div>
+			<div class="3"><img src="pilot.png"></div>
```
Comment (nacm1269) [inline index.html]: Could use more semantic naming here - "pilot-pic" for example

File: style.css [hunk 13]
```diff
@@ -5,39 +5,36 @@
 body {
 	background-color: #986960;
 	font-family: 'Raleway', sans-serif;
+	font-family: Raleway;
+	src: url("raleway");
+	background: linear-gradient(to bottom,
```
Comment (nacm1269) [inline style.css:10]: Sorry, I left the comment  re- background colour on the wrong css file

**Review by nacm1269 — COMMENTED** [review]
Looks good! Some of the comments towards the end may not be needed as you'll address as you get there.

**Comment by onekatytocode** [comment]
Thanks Nathan, some great points. I hope I have fixed most if not all.

**Pull request merged**